src/textual/css/_style_properties.py

```diff
@@ -60,9 +60,20 @@
 
 
 class GenericProperty(Generic[PropertyGetType, PropertySetType]):
-    def __init__(self, default: PropertyGetType, layout: bool = False) -> None:
+    """Descriptor that abstracts away common machinery for other style descriptors.
+
+    Args:
+        default: The default value (or a factory thereof) of the property.
+        layout: Whether to refresh the node layout on value change.
+        children: Whether to refresh the node children on value change.
+    """
+
+    def __init__(
+        self, default: PropertyGetType, layout: bool = False, children: bool = False
```

Comment:
I think maybe the name could use an update. I know its inherited from the refresh method, but in this context its not clear what it does. How about `refresh_children`?

Reply:
I actually like your point. Should I also change `layout` to `refresh_layout`?

Reply:
`layout` feels fine. It's an established term. It also works as a verb. It was just `children` felt too generic in context.